app.js

```diff
@@ -38,7 +38,7 @@ hamburger.addEventListener('click', function () {
         each.style.textDecoration = 'none';
         each.style.color = 'white';
         each.style.fontWeight = '700';
-        each.style.fontSize = '2.5rem';
+        each.style.fontSize = '2.0rem';
        })
 
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215685328-19fc23cb-a3b4-4702-8323-6c959020334f.png)

- [ ] You don't have the right Linters set for this pull request because ESLint is missing from your linters check which you can see in the image above :arrow_up: Kindly use these [linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) :+1: